test/test_user_settings.py

```diff
@@ -44,11 +49,21 @@ def setUpClass(self):
     os.environ["NOT_PARSED"] = "ignored"
 
 
-
   @classmethod
   def tearDownClass(self):
     os.chdir(self.working_dir)
 
+    # Other unittests might depend on defaults:
+    # Restore monkey patched settings ...
+    for key, val in self.settings_backup.iteritems():
+      setattr(in_toto.settings, key, val)
+
+    # ... and delete test environment variables
+    del os.environ["IN_TOTO_ARTIFACT_EXCLUDE_PATTERNS"]
+    del os.environ["IN_TOTO_ARTIFACT_BASE_PATH"]
+    del os.environ["IN_TOTO_not_whitelisted"]
```

Comment:
The rest of this variable is lowercase. is this intentional?

Reply:
Yes, to acknowledge case sensitivity in `user_settings.get_env`.

Reply:
ah, ok ok